classes/form/CustomerFormatter.php

```diff
@@ -209,7 +209,7 @@ public function getFormat()
                         'Shop.Forms.Labels'
                     )
                 )
-                ->addAvailableValue('placeholder', Tools::getDateFormat())
+                ->addAvailableValue('placeholder', $this->translator->trans('DD/MM/JJJJ', [], 'Shop.Theme.Customeraccount'))
```

Comment:
@okom3pom I think it should be `DD/MM/YYYY`?

Reply:
oups

Reply:
Fixed @kpodemski